sample-ldap/README.md

```diff
@@ -0,0 +1,84 @@
+# Overview
+
+This sample provides details on how an on-premise LDAP server can be accessed from an application running on SAP BTP, Kyma runtime.
+
+The sample uses SAP Connectivity Proxy and SAP Cloud Connector to establish connection to the on-premise LDAP server.
+
+In the sample flow, the users details are queried from the on-premise LDAP server via an HTTP GET call `/users/{userid}`.
+
+![flow](assets/kyma-ldap-on-prem.png)
+
+
+## Prerequisites
+
+* [SAP BTP, Kyma runtime instance](../prerequisites/#kyma)
+* [Kubernetes tooling](../prerequisites/#kubernetes)
+* [Cloud Connector on your laptop or test system](../prerequisites/#sap-cloud-connector)
+* [LDAP Server added as a virtual host in the cloud connector](https://help.sap.com/docs/CP_CONNECTIVITY/cca91383641e40ffbe03bdc78f00f681/f94810a111dd40fda07caffbbc02af87.html)
+
+## Set up
+
+### LDAP Server
+
+In my sample, I have used [ApacheDS](https://directory.apache.org/apacheds/) as an LDAP server and ran it on my laptop. You are free to use any other LDAP Server of your choice. I have added a user which I query via the API.
+
+![apache-ds](assets/apache-ds.png)
+
+### Kyma configuration
+
+* Set up environment variables
+
+  ```shell script
+  export NS={your-namespace}
+  ```
+
+* Create Connectivity Service Instance in any namespace.
+  >Note: Only one instance is required for the cluster. If you have deployed it previously, simply skip this step.
+
+  ![connectivity](./assets/connectivity-proxy.png)
+
+* Wait until the connectivity proxy is deployed. This can take around 10 minutes.
+
+  ```shell script
+  kubectl -n kyma-system get statefulsets.apps
+  ```
+  
+  You should see one entry for the connectivity-proxy stateful set
+
+  ```shell script
+  NAME                               READY   AGE
+  connectivity-proxy                 1/1     7d20h
+  ```
+
+* Expose the LDAP port via a service. Currently, LDAP port is not exposed by default.
+
+  ```shell script
+  kubectl -n kyma-system apply -f k8s/connectivity-proxy-ldap-service.yaml
+  ```
+
+* Create the secret with the required ldap configuration.
+
+  ```shell script
+  kubectl -n $NS create secret generic ldap-config --from-literal=LDAP_URL="ldap://{ldap-virtual-host}:{ldap-virtual-port}" \
```

Comment:
Does it make sense to have a short sentence on from where to fetch the differen tplaceholders?

Reply:
Updated PR